src/Vulkan/RHI/VkBuffer.cpp

```diff
@@ -83,14 +64,21 @@ VkBuffer::VkBuffer(NonNullPtr<VkGPUContext> ctx, const BufferDescription& desc)
 
     const ::vk::MemoryRequirements reqs = ctx->device.getBufferMemoryRequirements(*buffer);
 
+#if VEX_USE_CUSTOM_ALLOCATOR_BUFFERS
+    auto allocResult = allocator.AllocateResource(desc.memoryLocality, reqs);
```

Comment:
Nit: Could use a structured binding here to be more explicit than first, second.